src/web_crawler/twir_crawler.rs

```diff
@@ -188,7 +198,9 @@ impl TwirCrawler {
             .map(|element| self.extract_link_and_title(element))
             .filter(|issue| issue.title.contains(sentence))
             .collect();
-
+        
+        info!("{} link/s found on page {}", links.len(), origin_url);
```

Comment:
Please make it as a trace as well because it will be logged for every page. Please include the page as well. "Links found on page -> <which_page_title>"